src/rdb.h

```diff
@@ -103,6 +104,7 @@
 #define rdbIsObjectType(t) (((t) >= 0 && (t) <= 7) || ((t) >= 9 && (t) <= 21))
 
 /* Special RDB opcodes (saved/loaded with rdbSaveType/rdbLoadType). */
+#define RDB_OPCODE_SLOT_INFO  244   /* Individual slot info, such as slot id and size (cluster mode only). */
```

Comment:
I did try to increment RDB_VERSION. However, one of the test `unit/expire` fails due to encoded value comparison. 

```
*** [err]: All time-to-live(TTL) in commands are propagated as absolute timestamp in milliseconds in AOF in tests/unit/expire.tcl
Expected 'restore foo17 1694015989710 bar\f-È\} Lc^ ABSTTL' to match 'restore foo17 * {*} ABSTTL' (context: type source line 944 file /workplace/harkrisp/redis/tests/support/util.tcl cmd {assert_match [lindex $patterns $j] $data} proc ::assert_aof_content level 1)
```

Our validation is to do a `string match `on the following `{restore foo17 * {*} ABSTTL}`

RDB_VERSION - 11

```
restore foo17 1694014381977 {bar
                                aôù} ABSTTL
```

RDB_VERSION - 12

```
restore foo17 1694015989710 bar\f-È\} Lc^ ABSTTL
```

I don't understand tcl well. @oranagra are the curly bracket is a literal match or to compare list of string within it ?

Note: RDB version bump on unstable also leads to failure of this test.

Reply:
i don't know Tcl that well either.
what seems to happen is that a list in Tcl is usually represented as a chain of words in `{}`, and so is a sub-list.
in this case, either the dump payload no longer contains spaces so it doesn't need to be wrapped in `{}`, or because it contains s `}` char that gets escaped, it decided to escape more symbols and dropped the `{}` wrapping.
in any case, the solution (i tested it), is to change `{*}` with just `*`. i.e. `{restore foo6 * * ABSTTL}`

Reply:
I can confirm this seems right! `read_from_aof` uses `lappend` to build its result. When `lappend` appends a string that contains spaces, it wraps it in `{}` but now it doesn't seem to contain spaces anymore.

```TCL
$ tclsh
% set res {}
% lappend res {foo}
foo
% lappend res {bar}
foo bar
% lappend res {foo bar}
foo bar {foo bar}
```

Reply:
Thanks both of you @oranagra / @zuiderkwast. I think both `{restore foo6 * * ABSTTL}` and `{restore foo6 * ABSTTL}` have the same meaning. I will stick to what Oran proposed for better readability in the test.